+MonteCarlo/@ParticleWalker/private/one_dt.m

```diff
@@ -50,7 +50,9 @@
             dxdydz = dxdydz*(1-intersectInfoBB.t); % what's remaining
 
             % ENABLE if the particles cannot leave the bounding box
-            %dxdydz(:) = Geometry.reflect(dxdydz, intersectInfoBB.vertices);
+            if strcmp(substrate.boundary, 'reflect')
```

Comment:
Relies on substrate.boundary to return 'reflect' if we don't want periodic boundary conditions, so maybe this is not the most future-proof way of doing it. Consider updating in the future to a clearly-named boolean